liquidctl/config/load.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+import sys
+
+def get_config_files(file=None, appname='liquidctl', **kwargs):
+    """
+    This will get all the potential file paths on the system for where the config file
+    can be
+
+    This will also handle the -f overrides
+    """
+    files = []
+    if file:
+        files.append(file)
+
+    if sys.platform == 'win32':
+        files.append(os.path.join(os.getenv('APPDATA'), appname, 'config.toml'))
```

Comment:
May need to consider system-wide settings as well.

Reply:
I'm not familiar enough with windows to know what those folders would be, this is what cameup when I was googling where to put application configs on windows